release/run_e2e.sh

```diff
@@ -98,6 +98,8 @@ pip install -q -r requirements.txt
 pip install -q -U boto3 botocore
 git clone -b "${RAY_TEST_BRANCH}" "${RAY_TEST_REPO}" ~/ray
 
+rm -rf "${RELEASE_RESULTS_DIR-/tmp/ray_release_test_artifacts/}" || true
```

Comment:
Should this use `sudo`, and run inside the while loop?

Reply:
Ah good call, yep, moved it